python/loyalty_workflow.py

```diff
@@ -0,0 +1,135 @@
+import logging
+from datetime import timedelta
+
+from temporalio import activity, workflow
+from temporalio.common import WorkflowIDReusePolicy
+from temporalio.exceptions import WorkflowAlreadyStartedError
+from temporalio.workflow import ParentClosePolicy
+from temporalio.client import WorkflowHandle
+
+import email_strings
+from shared import *
+
+
+# Basic activity that logs and does string concatenation
+@activity.defn(name="SendEmail")
```

Comment:
Why change the name from the default here? I wouldn't think matching histories across languages is enough of a reason to override what is a very reasonable default here. Same with other name overrides. We should only encourage users to override the name if there's a good reason.